database/studyu-schema.sql

```diff
@@ -491,6 +491,9 @@ ALTER FUNCTION public.user_email(user_id uuid) OWNER TO postgres;
 
 CREATE TABLE public.app_config (
     id text NOT NULL,
+    app_min_version text NOT NULL DEFAULT '0.0.0',
```

Comment:
I think we should not set a default here, but require this field to be set on table creation. The recommended "default value" would then be the current app version or the lowest version without a breaking database change. We will need to keep track of documenting app_min_version somewhere when we release.

Reply:
Fixed in https://github.com/hpi-studyu/studyu/pull/576/commits/8d36dc78771d748483bbf3ad22794a95e73dad06